docs/README.md

```diff
@@ -3,6 +3,12 @@
 [![Slack](https://img.shields.io/badge/slack-@cncf/otel--dotnet--auto--instr-brightgreen.svg?logo=slack)](https://cloud-native.slack.com/archives/C01NR1YLSE7)
 [![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.AutoInstrumentation.svg)](https://www.nuget.org/packages/OpenTelemetry.AutoInstrumentation)
 [![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.AutoInstrumentation.svg)](https://www.nuget.org/packages/OpenTelemetry.AutoInstrumentation)
+<a href="https://actuated.dev/">
+  <img
+    alt="Arm CI sponsored by Actuated"
+    src="https://docs.actuated.dev/images/actuated-badge.png" width="120px">
+  </img>
+</a>
```

Comment:
Tracked this syntax on: https://github.com/self-actuated/docs.actuated.dev/issues/18

Reply:
Badge is merged 👍 I'll update the docs also in the current branch tomorrow.

Reply:
```suggestion
[![Arm CI sponsored by Actuated](https://img.shields.io/badge/SA_actuated.dev-004BDD)](https://actuated.dev/)
```

Great!